packages/firebase_storage/lib/firebase_storage.dart

```diff
@@ -52,6 +52,8 @@ class StorageReference {
     return FirebaseStorage._channel.invokeMethod("StorageReference#delete",
         <String, String>{'path': _pathComponents.join("/")});
   }
+
+  String getPath() => _pathComponents.join("/");
```

Comment:
Given that we already have `child` and not `getChild`, we should maybe call it `path`. It could be made a simple getter like so:
```dart
/// The path of this reference.
String get path => _pathComponents.join('/');
```

Reply:
I'll fix that, bump the version and add a changelog!